.github/workflows/size.yml

```diff
@@ -11,11 +11,23 @@ jobs:
     steps:
       - name: Setup git
         uses: actions/checkout@v2
+        
       - name: Setup node
         uses: actions/setup-node@v1
         with:
           node-version: "16.6.1"
-      - run: npx lerna bootstrap
+
+      - name: Cache dependencies
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+      
+      - name: Install dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: npx lerna bootstrap
```

Comment:
This should be replaced with `yarn`, I think

Reply:
@denniskigen  is this possible?